### PR TITLE
Conver to md 1 to 5

### PR DESCRIPTION
**Pull request by jdvr**
The most changed post is "_GLOBAL CODE RETREAT 2011 IN BERLIN_" I have remove uppercase letters from title, I also have remove _further reading_ and includes as links on the text because I didn't find a way to make link to a list in same file.

The rest of post are almost identical. Just change html tag by md  and add/remove line breaks.

### CODE REVIEW

File: _posts/2011-12-04-coderetreat---the-need-for-preparedness.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+author: Mark Simpson
+authorname: Mark Simpson
+title: Coderetreat - the need for preparedness
+preview: (crossposted from verdammelt.posterous.com)
```
Comment (rradczewski) [inline _posts/2011-12-04-coderetreat---the-need-for-preparedness.md]: That's a rather bad preview, can you replace this with the abstract?

File: _posts/2011-12-04-global-code-retreat-2011-in-berlin.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+author: Matthias Günther
+authorname: Matthias Günther
+title: Global Code Retreat 2011 in Berlin
+preview: I attended the Global Coderetreat event initialized by Corey Haines2. The general purpose of this event was to get a bunch of hackers together to implement Conway’s Game of Life.
```
Comment (rradczewski) [inline _posts/2011-12-04-global-code-retreat-2011-in-berlin.md]: Theres a `2` next to Corey Haines

File: _posts/2011-12-04-global-code-retreat-2011-in-berlin.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+I attended the [**Global Coderetreat**](http://coderetreat.org/ "coderetreat") event initialized by [Corey Haines](http://coreyhaines.com/ "Corey Haines"). The general purpose of this event was to get a bunch of hackers together to implement [Conway’s Game of Life](http://en.wikipedia.org/wiki/Conway%27s_Game_of_Life "Conway's Game of Life").
+
+![](http://farm8.staticflickr.com/7029/6452198253_ca6e01b7f4.jpg "Coderetreat@Immobilienscout alt")
```
Comment (rradczewski) [inline _posts/2011-12-04-global-code-retreat-2011-in-berlin.md]: `Here goes the alt-text`

File: _posts/2011-12-04-global-code-retreat-2011-in-berlin.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Setting up goals
+
+As **J. B. Rainsberger** mentioned in his article about [four elements of simple design](http://www.jbrains.ca/permalink/the-four-elements-of-simple-design "four elements of simple design"), it is best to setup goals for your retreat to get most out of it. Here is a list of what I achieved in every section:
+
+1.  Being a mentor of TDD: I taught [dedan](https://github.com/dedan "dedan") how to use TDD, why  
+    TDD helps you to think about your design
+2.  Being a mentor of BDD: Showing a fellow Rubyist how to do BDD
+3.  Doing something completely stupid: I played _evil coder_ and _ping-pong pairing_ with  
+    [@tilosoenke](http://twitter.com/#%21/tilosoenke "@tilosoenke") - we couldn’t communicate with
```
Comment (rradczewski) [inline _posts/2011-12-04-global-code-retreat-2011-in-berlin.md]: That link is broken (and the `alt` extra is redundant). Should be `https://twitter.com/tilosoenke`

File: _posts/2011-12-04-global-code-retreat-2011-in-berlin.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+1.  Being a mentor of TDD: I taught [dedan](https://github.com/dedan "dedan") how to use TDD, why  
+    TDD helps you to think about your design
+2.  Being a mentor of BDD: Showing a fellow Rubyist how to do BDD
+3.  Doing something completely stupid: I played _evil coder_ and _ping-pong pairing_ with  
+    [@tilosoenke](http://twitter.com/#%21/tilosoenke "@tilosoenke") - we couldn’t communicate with  
+    words, only with code, one writing tests and other the implementation of the tests.
+4.  Discuss about names: We said together and discussed about right names for methods, variables, and  
+    classes. We concluded that good names saves you scary comments in your code
+5.  Explain Ruby to a Java-Programmer
+6.  Being mentored about Lisp: We solved the whole ‘Game of Live’ in common [Lisp](http://en.wikipedia.org/wiki/Lisp_%28programming_language%29 "Lisp"), while
```
Comment (rradczewski) [inline _posts/2011-12-04-global-code-retreat-2011-in-berlin.md]: Same here, extra `"Lisp"`

File: _posts/2011-12-04-coderetreat-2011-in-prague.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+The paradox is that by solving a trivial problem one can try out such a lot of things in a few hours - it just need to have a focus and not to have a target. A big surprise for me today was DART - along with [@kolman](https://twitter.com/#!/kolman) I spent some time playing with the problem in DART language (by the way thanks for the energy devoted to setting development environment up) and we came to the conclusion that DART is essentially a mix of C# and Java. Given lack of documentation when we weren't able to find out the notation we tried C# syntax and when it fails to compile we exchanged it for Java notation (@kolman is .NET guy and I'm from Java world). Compiler usually accepted one of the options we tried. I'm just wondering what audience Google aims to - C# programmers or Java ones? May be both?
```
Comment (rradczewski) [inline _posts/2011-12-04-coderetreat-2011-in-prague.md]: Permalinks for twitter should def. be fixed (see below for an example)

File: _posts/2011-12-04-coderetreat---the-need-for-preparedness.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+## An Initial Attempt for a Personal Solution
+
+Given that I saw (and felt, when pairing with someone who was  
+unprepared) this problem at the Chicago coderetreat (held the day after  
+[SCNA](http://scna.softwarecraftsmanship.org/)), I made sure I had a ruby and Java project all set up so  
+that I had a failing test (`true == false`). That way I knew that  
+everything was all set: all needed libraries, any special configuration  
+([autotest](http://www.zenspider.com/ZSS/Products/ZenTest/) & [rspec](https://www.relishapp.com/rspec) always gives me a little bit of  
+hassle), any other annoying little thing was done. I even went so far  
+as to push them up to [GitHub](https://.github.com) so others could use them to.
```
Comment (rradczewski) [inline _posts/2011-12-04-coderetreat---the-need-for-preparedness.md]: Link is broken

File: _posts/2011-12-02-on-the-role-of-the-coderetreat-facilitator.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+<object width="400" height="225" classid="clsid:d27cdb6e-ae6d-11cf-96b8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,40,0" _origwidth="400"><param name="allowfullscreen" value="false"><param name="allowscriptaccess" value="never"><param name="wmode" value="opaque"><param name="src" value="http://vimeo.com/moogaloop.swf?clip_id=18955165&amp;server=vimeo.com&amp;show_title=0&amp;show_byline=0&amp;show_portrait=0&amp;color=00adef&amp;fullscreen=1&amp;autoplay=0&amp;loop=0"><embed wmode="opaque" width="400" height="225" type="application/x-shockwave-flash" src="http://vimeo.com/moogaloop.swf?clip_id=18955165&amp;server=vimeo.com&amp;show_title=0&amp;show_byline=0&amp;show_portrait=0&amp;color=00adef&amp;fullscreen=1&amp;autoplay=0&amp;loop=0" allowfullscreen="false" allowscriptaccess="never" _origwidth="400"></object>
```
Comment (rradczewski) [inline _posts/2011-12-02-on-the-role-of-the-coderetreat-facilitator.md]: Embedding seems to be broken on this. Have you looked into how we could fix this? Vimeo probably has a JavaScript/HTML5 solution for this I imagine

Comment (jdvr) [inline _posts/2011-12-02-on-the-role-of-the-coderetreat-facilitator.md]: I have included an iframe, this doesn't require flash

**Comment by jdvr** [comment]
I think this PR is ready to merge or close.

**Pull request merged**